draft-ietf-httpbis-safe-method-w-body.xml

```diff
@@ -201,6 +201,17 @@ q=foo&amp;limit=10&amp;sort=-published
       QUERY responses are illustrated in <xref target="examples" />.
     </t>
 
+    <t>
+      A successful response to a QUERY request &MAY; include a Content-Location
+      header field (see <xref target="HTTP" section="8.7"/>) containing an
+      identifier for a specific resource corresponding to the results of the
+      operation. A client can send a GET request for the indicated URI to repeat
```

Comment:
```suggestion
      operation. A client can send a GET request for the indicated URI to repeat
```

So is this "A client can" or "This represents a claim from the server that a client can"